Procfile

```diff
@@ -1,5 +1,5 @@
-a1: ./alertmanager --log.level=debug --storage.path=$TMPDIR/a1 --web.listen-address=:9093 --mesh.peer-id=00:00:00:00:00:01 --mesh.nickname=a --mesh.listen-address=:8001 --config.file=examples/ha/alertmanager.yaml
-a2: ./alertmanager --log.level=debug --storage.path=$TMPDIR/a2 --web.listen-address=:9094 --mesh.peer-id=00:00:00:00:00:02 --mesh.nickname=b --mesh.listen-address=:8002 --mesh.peer=127.0.0.1:8001 --config.file=examples/ha/alertmanager.yaml
-a3: ./alertmanager --log.level=debug --storage.path=$TMPDIR/a3 --web.listen-address=:9095 --mesh.peer-id=00:00:00:00:00:03 --mesh.nickname=c --mesh.listen-address=:8003 --mesh.peer=127.0.0.1:8001 --config.file=examples/ha/alertmanager.yaml
+a1: ./alertmanager --log.level=debug --storage.path=$TMPDIR/a1 --web.listen-address=:9093  --cluster.address=127.0.0.1:8001 --config.file=examples/ha/alertmanager.yaml
```

Comment:
Do you think we should keep the flags, for backward compatibility?

Reply:
Mh, we could add a hidden flag with the old name. But some flags were removed entirely – we could just blackhole these but then people will have mis-aligned expectations and we might get complaints.

In particular I removed the ability to set a password. Memberlist provides similar functionality – but our general policy on security features is that we aren't able to support them. I felt this is a similar case and it would be better to be consistent. /cc @brian-brazil 

Ultimately, people have to switch anyway. As it's not a 1:1 mapping, maybe it's better to be explicit and just fail hard?

Reply:
Makes sense. In that case I'm definitely for a beta/rc release though, just so people get the notion that larger things are changing here.

Reply:
I'm a little unclear exactly what the change is here from the PR description, if we have to break I'm generally in favour of hard breaks as it's easier to debug.

For consistency we should keep away from security features and I'm unsure exactly what properties this password offered. When Ben's TLS plans come to fruition we can reconsider, which will be a different approach anyway.

Reply:
Basically it's about encrypting the peer-to-peer traffic with a secret. FWIW, I don't think that's really needed for a while.

Reply:
Any idea what that might look like? Seems like TLS is not really a natural fit for the way memberlist communication works. Maybe we should think this through a bit more just to make sure we're not setting ourselves road blocks for the future in this regard.

Reply:
Encryption isn't much use without signing. TLS is fine for it I'd think, every cluster endpoint would have a cert signed by some CA and they'd be checked at at connection time. ServerName might be a bit interesting here if we only have IPs (not so much a problem if we use service discovery), but it's in line with the way @SuperQ and I have been talking about TLS.

Reply:
In mesh and memberlist alike it is a builtin feature – we can take it or leave it

Reply:
It's not in line with future plans (and a secret on the command line is a bad idea anyway), so I'd say remove it while we're breaking things.

Reply:
> It's not in line with future plans (and a secret on the command line is a bad idea anyway), so I'd say remove it while we're breaking things.

I disagree: A secret on the commandline is more secure than an unencrypted message that is flowing unencrypted between 2 datacenters.

For the next feature that you drop: please put it in the release notes. This was hard to find.